azure-pipelines.yml

```diff
@@ -34,13 +34,12 @@ jobs:
     - job: Code_check
       displayName: Code check
       pool:
-        # agent pool can't be read from a user-defined variable (Azure DevOps limitation)
         ${{ if eq(variables['System.TeamProject'], 'public') }}:
           name: NetCorePublic-Int-Pool
-          queue: BuildPool.Windows.10.Amd64.VS2019.BT.Open
+          queue: BuildPool.Windows.10.Amd64.VS2019.Open
```

Comment:
Why change the Code Check job? Has been fine using the BuildTools queues.

Reply:
For the sake of consistency.